backstory/src/main/java/com/google/sps/servlets/StoryManagerRequestFactoryImpl.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.api.client.http.ByteArrayContent;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.IdTokenCredentials;
+import com.google.auth.oauth2.IdTokenProvider;
+import java.io.IOException;
+
+/**
+ * Object for building POST Requests to a GPT-2 Container.
+ */
+
+public class StoryManagerRequestFactoryImpl implements StoryManagerRequestFactory {
+  private String requestBody;
+
+  /**
+   * Builds a PostRequest given parameters.
+   *
+   * @return HttpRequest Post Request
+   */
+  public HttpRequest buildPostRequest(String requestBody) throws IOException {
```

Comment:
Can you describe what the format `requestBody` should have? For example, is it JSON? If so, are there particular parameters that are required?